src/dmd/audit_log.d

```diff
@@ -0,0 +1,82 @@
+/**
+ * Generate text files for assistance in debugging dmd.
+ *
+ */
+
+module dmd.audit_log;
+import core.stdc.stdlib;
+import core.stdc.stdio;
+import dmd.root.rootobject;
+import dmd.globals;
+import dmd.ast_node;
+import dmd.hdrgen;
+import dmd.expression;
+import dmd.dsymbol;
+import dmd.statement;
+import dmd.mtype;
+import dmd.init;
+import dmd.dtemplate;
+import dmd.common.outbuffer;
+
+private FILE*[string] file;
+
+void Audit_Log(ASTNode node, string functioncall = __FUNCTION__, string modulecall = __MODULE__)
+{
+    debug(AUDITLOG)
```

Comment:
What's the point of putting the entire body of the function in a debug statement since you are already enabling this feature through the CLI?